packages/components/src/components/select-wrapper/select-wrapper-dropdown/select-wrapper-dropdown-utils.spec.ts

```diff
@@ -342,3 +343,69 @@ describe('getDropdownVisibility()', () => {
     expect(getDropdownVisibility(isOpen, type, resetFilter)).toBe(result);
   });
 });
+
+describe('updateNativePopoverSelectStyles()', () => {
+  const optionMaps: OptionMap[] = generateOptionMaps();
+  const nativePopover = document.createElement('div');
+  const nativeButton = document.createElement('button');
+  const host = document.createElement('div');
+
+  const setViewport = () => {
+    // clientWidth/Height mockBoundingClientRect is always 0 in JSDOM so we mock it
+    Object.defineProperties(document.documentElement, {
+      clientWidth: {
+        value: 1000,
+        configurable: true,
+      },
+      clientHeight: {
+        value: 1000,
+        configurable: true,
+      },
+    });
+  };
+  beforeAll(() => {
+    setViewport();
+  });
+
+  beforeEach(() => {
+    jest.spyOn(nativeButton, 'getBoundingClientRect').mockImplementation(
+      () =>
+        ({
+          height: 30,
+          width: 100,
+          left: 10,
+          top: 20,
+          bottom: 50,
+          right: 110,
+        }) as DOMRect
+    );
+  });
+
+  afterEach(() => {
+    jest.clearAllMocks();
```

Comment:
I think that's covered by `jest.config.js`.

Reply:
Fixed